modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessExecuteAction.java

```diff
@@ -175,6 +181,8 @@ static class ContextSetup implements Writeable, ToXContentObject {
                 );
             }
 
+            @Nullable // null means local cluster
+            private final String clusterAlias;  // this field is not Writeable, as it is needed only on the initial receiving node
```

Comment:
just double checking: the index that was provided is not printed out anywhere in the output, as we only return the result of the script execution?

Reply:
Correct. The output of this endpoint is just:
```
{
  "result": true
}
```
where `true` is the value and can be any object (number, string, bool, etc).

Reply:
ok we are good then, i was worried about cases that we have in search, that need to preserve the original index name including the cluster alias to output it as part of the response. Glad we don't have this problem in painless execute.